### DISCUSSION TITLE
/getting-started/directory-structure.html

### DISCUSSION THREAD

**Discussion by giscus** [goravel/goravel#349] [Announcements]
# /getting-started/directory-structure.html

[[toc]] Introduction The default file structure can make you better start project advancement, and you can also add new folders freely, but do not modify the default folders. Ro...

https://www.goravel.dev/getting-started/directory-structure.html

**Comment by justwannabecracked** [comment]
I love this docs, short and simple

**Comment by arhinful** [comment]
Interesting 🔥

**Comment by GontangRagilPrakasa** [comment]
love it
